docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -0,0 +1,238 @@
+
+### `profile.godeltaprof_block`
+
+Collects profiles from godeltaprof block endpoint. The delta is computed on the target.
+
+| Argument | Type    | Description                                | Default                   | Required |
+| -------- | ------- | ------------------------------------------ | ------------------------- | -------- |
+| enabled  | boolean | Enable this profile type to be scraped.    | false                     | no       |
+| path     | string  | The path to the profile type on the target. | "/debug/pprof/delta_block" | no       |
+
+## Example
+
+The following configuration sets up a scraping job that scrapes two local applications (the Agent itself and Pyroscope). The profiles obtained are then sent over to the receivers as defined by other components.
+
+```river
+pyroscope.scrape "default_settings" {
+  targets    = ["http://localhost:12345"]
+  forward_to = ["http://localhost:4040"]
+  profiling_config {
+    profile.goroutine {
+      enabled = true
+      path = "/debug/pprof/goroutine"
+      delta = false
+    }
+    profile.mutex {
+      enabled = true
+      path = "/debug/pprof/mutex"
+      delta = false
+    }
+    profile.process_cpu {
+      enabled = true
+      path = "/debug/pprof/profile"
+      delta = true
+    }
+    profile.godeltaprof_memory {
+      enabled = false
+      path = "/debug/pprof/delta_heap"
+    }
+    profile.memory {
+      enabled = true
+      path = "/debug/pprof/memory"
+      delta = false
+    }
+    profile.godeltaprof_mutex {
+      enabled = false
+      path = "/debug/pprof/delta_mutex"
+    }
+    profile.godeltaprof_block {
+      enabled = false
+      path = "/debug/pprof/delta_block"
+    }
+    profile.block {
+      enabled = true
+      path = "/debug/pprof/block"
+      delta = false
+    }
+  }
+}
+
+```
+
+By adjusting the configuration as demonstrated above, you can enable the Grafana Agent to suitably scrape and process profile data from your pprof endpoints.
+
+
+Copy the example configuration files from the [Pyroscope example](https://github.com/grafana/pyroscope/tree/main/examples/grafana-agent) to their respective locations in the project directory:
+
+- `examples/your-application-example/agent/config/config.river`
+- `examples/your-application-example/docker-compose.yml`
+- `examples/your-application-example/pyroscope/config.yaml`
+
+Open the `examples/your-application-example/agent/config/config.river` file and configure the `pyroscope.write` section based on your needs. 
+
+**Ensure that the url property points to the correct Pyroscope instance.**
+
+Open the `examples/your-application-example/docker-compose.yml` file and review the service definitions for pyroscope and agent. Adjust the volumes and ports if needed.
+
+Open the `examples/your-application-example/pyroscope/config.yaml` file and modify the `scrape_configs` section as required. Add the necessary targets for your Go application.
```

Comment:
There is no need to use both agent scraping and pyroscope scraping at the same time. They do same thing just from different places. I think we should remove `pyroscope/config.yaml` from this doc.